src/PublishPort.jsx

```diff
@@ -47,18 +93,31 @@
                             min={1}
                             max={65535}
                             value={item.hostPort || ''}
-                            onChange={(_event, value) => onChange(idx, 'hostPort', value)} />
+                            validated={validationFailed?.hostPort ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20231016-121055-6688b0e7-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)